ui/imports/shared/popups/send/controls/GasSelector.qml

```diff
@@ -16,18 +16,19 @@ Item {
 
     property var selectedAsset
     property string currentCurrency
-
     property var bestRoutes
```

Comment:
It's a model, but it's not clear without reading impl details. It would be handy to have expected roles described.